addons/im_livechat/static/src/models/messaging_initializer/messaging_initializer.js

```diff
@@ -26,7 +26,7 @@ registerInstancePatchModel('mail.messaging_initializer', 'im_livechat/static/src
             // flux specific: channels received at init have to be
             // considered pinned. task-2284357
             if (!channel.isPinned) {
-                channel.update({ isPendingPinned: true });
+                this.setIsPinned(true);
```

Comment:
I guess it should be `channel.pin()`?

Reply:
Indeed, used to limit my search to `mail` module, my bad